react/ProductLink.tsx

```diff
@@ -43,10 +45,23 @@ export default function ProductLink(props: Props) {
     setShouldReplaceUrl(Boolean(modalDispatch))
   }, [modalDispatch])
 
-  return (
+  return target && target !== '_self' ? (
+    <a target={target} href={resolvedLink} className={handles.link}>
+      {label && <span className={handles.label}>{label}</span>}
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
+    </a>
+  ) : (
     <Link to={resolvedLink} className={handles.link} replace={shouldReplaceUrl}>
       {label && <span className={handles.label}>{label}</span>}
-      {children && <div className={handles.childrenContainer}>{children}</div>}
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
     </Link>
   )
```

Comment:
I guess this logic is not needed anymore after https://github.com/vtex-apps/render-runtime/pull/486